pages/tokens/reset_password.php

```diff
@@ -0,0 +1,37 @@
+<?php
+$token = AccessToken::retrieve($_GET['token'] ?? "");
+page("Réinitialiser le mot de passe")->disableNav()->heading(false);
```

Comment:
For clarity, I would rather move this down closer to the rendered content :) e.g. line 19